cms/templates/js/course-outline-modal.underscore

```diff
@@ -1,14 +1,6 @@
-<%
-var enable_proctored_exams = enable_proctored_exams;
-var enable_timed_exams = enable_timed_exams;
-%>
-
 <div class="xblock-editor" data-locator="<%= xblockInfo.get('id') %>" data-course-key="<%= xblockInfo.get('courseKey') %>">
-    <% if (!( enable_proctored_exams || enable_timed_exams )) { %>
-        <div class="message modal-introduction">
-            <p><%- introductionMessage %></p>
-        </div>
-    <% } %>
+    <div class="message modal-introduction">
+        <p><%- introductionMessage %></p>
```

Comment:
Sorry for the back and forth, @douglashall and @afzaledx. Looking at the updated sandbox, I think we can remove the "Change the settings for ..." text. Its not needed given the rest of the UI's context. Thanks for the attention to minor details!